src/hooks/useClientSideV3Trade.ts

```diff
@@ -23,6 +23,7 @@ const DEFAULT_GAS_QUOTE = 2_000_000
  * @param amountSpecified the exact amount to swap in/out
  * @param otherCurrency the desired output/payment currency
  */
+// TODO: delete this function & deprecate it after swapping out USDC usage for SOR
```

Comment:
will do this in a separate PR

Reply:
> after swapping out USDC usage for SOR

What is this referring to?

Reply:
`useUSDCPrice` still uses useV2Quoter & useV3Quoter to calculate the price in USDC of the input currency - but now that we have a multichain SOR, we can deprecate those in favor of using SOR

Reply:
https://github.com/Uniswap/widgets/pull/58